extensions/2.0/Vendor/MSFT_CollisionPrimitives/README.md

```diff
@@ -0,0 +1,75 @@
+# MSFT\_CollisionPrimitives
+
+## Status
+
+Draft
+
+## Dependencies
+
+Written against glTF 2.0 spec.
+
+## Overview
+
+This extension adds the ability to specify collision primitives inside a glTF asset. This extension does not mandate any particular behaviour for those objects aside from their collision geometry. These types may be used in combination with additional extensions or application-specific business logic. The MSFT\_RigidBodies extension uses primitives defined in this spec to provide geometric primitives for collision detection in a rigid body simulation.
+
+## glTF Schema Updates
+
+The `MSFT_CollisionPrimitives` extension can be added to any `node` to define one or more of the following properties:
+
+| |Type|Description|
+|-|-|-|
+|**collider**|`object`|Create a collision shape in the physics engine for this node.|
```

Comment:
Is this meant to be an `integer` indexing into `colliders` with an accompanying `node.MSFT_CollisionPrimitive.schema.json`?

Reply:
Ah, yes; had some incorrect filenames, have renamed them and tightened up the wording on the README, so it's clearer.